pkg/test/environment.go

```diff
@@ -1020,6 +1022,62 @@ func (e *Environment) StartMockKubernetesMetricServer(ctx context.Context) (port
 	return port
 }
 
+func (e *Environment) StartAgentDisconnectServer(ctx context.Context) int {
+	var dPort int
+	if port := os.Getenv("AGENT_DISCONNECT_PORT"); port == "" {
```

Comment:
Configure this with the other ports above

Reply:
Done